addons/website_slides/static/src/xml/website_slides_share.xml

```diff
@@ -67,7 +67,7 @@
                         <i class="fa fa-envelope-o"/> Send Email
                     </button>
                 </div>
-                <div class="alert alert-info d-none" role="alert"><strong>Sharing is caring!</strong> Email(s) sent. </div>
+                <div class="alert alert-info d-none" role="alert"><strong>Sharing is caring!</strong> Your email(s) will arrive shortly.</div>
```

Comment:
We never fix labels in old stables, only stack traces and issues breaking actual flows.

Let's re-target to 18.0 if you don't mind :slightly_smiling_face: